### ISSUE TITLE
Left option as meta does not work in psql

### ISSUE THREAD

**Issue by generalpiston** [warpdotdev/Warp#2796]
### Describe the bug

When using `psql` with `Left option as meta`, I expect that I can move forward 1 word and backwards 1 word using Option-f and Option-b, respectively. It does not do that. Instead I get `ƒ` and `∫`.

### To Reproduce

1. Open Warp.
2. Start `psql`.
3. Type `Option-f` or `Option-b`.

### Expected behaviour

It should go forward /backwards 1 word .

### Screenshots

<img width="288" alt="image" src="https://user-images.githubusercontent.com/2106636/226280889-40ab8b18-462f-432a-8e09-c2762b6fc842.png">
<img width="305" alt="image" src="https://user-images.githubusercontent.com/2106636/226280923-7c06c9cc-fd9e-4e09-8584-2f6cb84409ef.png">
<img width="502" alt="image" src="https://user-images.githubusercontent.com/2106636/226280985-58fe480e-c37d-4645-8469-ee72b946c4a4.png">

### Operating System

MacOS

### OS Version

13.2.1

### Shell Version

zsh 5.8.1 (x86_64-apple-darwin22.0)

### Warp Version

v0.2023.03.07.08.02.stable_03

### Additional context

It works outside of `psql`.

### Does this block you from using Warp daily?

No

### Warp Internal (ignore): linear-label:b8107fdf-ba31-488d-b103-d271c89cac3e

None

**Comment by dannyneira** [comment]
Thanks for letting us know @generalpiston 
We're tracking this issue here #64

TLDR, it's something that we're working on, but tricky due to the way Warp handles input is completely different from other terminals ( works like code editor ).

Please :+1: and comment there if you're also seeing the issue, as it helps us gauge the impact of the bug.

Closing as duplicate.

**Comment by Ryokki** [comment]
I tried a lot of cli tools (psql, mysql, redis-cli) and found that all of them have this problem, which bothers me a lot. Looking forward to your solution to this problem
